index.d.ts

```diff
@@ -101,6 +107,11 @@ declare namespace aperture {
     Returns a `Promise` for the path to the screen recording file.
     */
     stopRecording: () => Promise<string>;
+
+    /**
+     Cancels a recording, if it was started
+     */
+    cancelRecording: () => void;
```

Comment:
I just noticed that we have been very inconsistent with naming. I think this should be `cancel()`. I'll fix the other methods when this is merged.